app.py

```diff
@@ -112,6 +112,12 @@ def serve_layout():
     const {min, max, colorscale, circleOptions, colorProp} = context.props.hideout;
     const csc = chroma.scale(colorscale).domain([min, max]);
     circleOptions.fillColor = csc(feature.properties[colorProp]);
+    // Lower opacity for sensors with stale data
+    if (new Date() - new Date(feature.properties._time) > 1000 * 60 * 60 * 2) {
```

Comment:
I think it might be nice to move these numbers that define the staleness threshold into a variable that's defined at the top of the app.

Reply:
Yeah that's a good idea. #90 moves JS code out into `.js` files, so let's do this on top of that.